packages/angular_devkit/build_angular/src/builders/application/execute-build.ts

```diff
@@ -225,53 +211,75 @@ export async function executeBuild(
     estimatedTransferSizes = await calculateEstimatedTransferSizes(executionResult.outputFiles);
   }
 
-  // Perform i18n translation inlining if enabled
-  let prerenderedRoutes: string[];
-  let errors: string[];
-  let warnings: string[];
-  if (i18nOptions.shouldInline) {
-    const result = await inlineI18n(options, executionResult, initialFiles);
-    errors = result.errors;
-    warnings = result.warnings;
-    prerenderedRoutes = result.prerenderedRoutes;
-  } else {
-    const result = await executePostBundleSteps(
-      options,
-      executionResult.outputFiles,
-      executionResult.assetFiles,
-      initialFiles,
-      // Set lang attribute to the defined source locale if present
-      i18nOptions.hasDefinedSourceLocale ? i18nOptions.sourceLocale : undefined,
-    );
-
-    errors = result.errors;
-    warnings = result.warnings;
-    prerenderedRoutes = result.prerenderedRoutes;
-    executionResult.outputFiles.push(...result.additionalOutputFiles);
-    executionResult.assetFiles.push(...result.additionalAssets);
-  }
+  if (executionResult.errors.length === 0) {
```

Comment:
Should be able to drop this conditional. Bundle budget errors won't cause anything else to break and might as well get a comprehensive list of errors when possible.
If there are bundling errors which are typically fatal to the rest of the steps, the build already exits early.